client/src/views/pages/you-are-banned-what-now.js

```diff
@@ -0,0 +1,486 @@
+
+import steamAvatar from '../../assets/img/misc/avatar.jpg';
+
+import SteamUserSearchBox from '../../components/steam-user-search-box-wizard.js';
+
+import {
+    AdvancedModal
+  } from '../../components';
+
+
+const GET_PLAYER = gql`
+  query Search($id: String!) {
+    steamUser(id: $id) {
+      id
+      name
+      avatarFull
+      reputationPoints
+      riskRating
+      reputationRank
+      lastRefreshedInfo
+      lastRefreshedReputationPoints
+      lastRefreshedReputationRank
+      activeBans: bans(orderBy: "created", orderDirection: DESC, expired: false) {
+        edges {
+          cursor
+          node {
+            id
+            banList {
+              id
+              name
+              organisation {
+                id
+                name
+                discord
+              }
+            }
+            reason
+            created
+            expires
+          }
+        }
+      }
+      expiredBans: bans(orderBy: "created", orderDirection: DESC, expired: true) {
+        edges {
+          cursor
+          node {
+            id
+            banList {
+              id
+              name
+              organisation {
+                id
+                name
+                discord
+              }
+            }
+            reason
+            created
+            expires
+          }
+        }
+      }
+    }
+  }
+`;
+
+const Wizard = (props) => {
+    const [state, updateState] = useState({
+        form: {},
+         outSideControls: false,
+    });
+
+    const updateForm = (key, value) => {
+        const { form } = state;
+
+        form[key] = value;
+        updateState({
+            ...state,
+            form,
+        });
+    };
+
+    // Do something on step change
+    const onStepChange = (stats) => {
+        //console.log(stats);
+    };
+
+    const setInstance = SW => updateState({
+        ...state,
+        SW,
+    });
+
+    return (
+        <div className='container'>
+            <div className={'jumbotron mt-4'}>
+                <div className='row'>
+                    <div className={`col-12 col-sm-6 offset-sm-3 ${styles['rsw-wrapper']}`}>
+                        <StepWizard
+                            onStepChange={onStepChange}
+                            isHashEnabled
+                            instance={setInstance}
+                        >
+                            <First hashKey={'FirstStep'} update={updateForm} steamUser={props.steamUser}/>
+                            <Second hashKey={'SecondStep'} form={state.form} steamUser={props.steamUser}/>
+                            <Third hashKey={'ThirdStep'} update={updateForm} steamUser={props.steamUser} />
+                            <Fourth hashKey={'FourthStep'} update={updateForm} steamUser={props.steamUser} />
+                            <Fifth hashKey={'FifthStep'} update={updateForm} steamUser={props.steamUser} />
+                            <Last hashKey={'LastStep'} />
```

Comment:
Have you thought about storing the components in an array and then passing the `hashKey` as an index of the slide in the array?